anitya/lib/backends/__init__.py

```diff
@@ -30,82 +30,9 @@
 import anitya
 import anitya.app
 from anitya.lib.exceptions import AnityaPluginException
+from anitya.lib.versions import RpmVersion
 import six
 
-try:
-    from rpm import labelCompare as _compare_rpm_labels
-except ImportError:
-    # Emulate RPM field comparisons as described in
-    # http://stackoverflow.com/questions/3206319/how-do-i-compare-rpm-versions-in-python/3206477#3206477
```

Comment:
Plus we only use it to let the test suite pass in the absence of the RPM C bindings, which is common in local testing under tox, but shouldn't happen in a production deployment using the system site-packages.